src/Utils/Vector.cpp

```diff
@@ -0,0 +1,43 @@
+//
+// Created by lisandro on 24/10/20.
+//
+
+#include "Vector.h"
+
+int Vector::getX() { return xPosition;}
+
+int Vector::getY() { return yPosition;}
+
+void Vector::setX(int x) { xPosition = x;}
```

Comment:
Not smt to fix, but to bear in mind only, instead of doing this set, you could have done a setVector(int x, int y), and that would have been fine (this is not bad per se, but a little redundant maybe).